src/web/app/components/panel-chevron/panel-chevron.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="chevron">
+<button type="button" class="chevron btn-course btn btn-primary btn-sm" aria-label="Open table with sessions for this course" [attr.aria-expanded]=isExpanded>
```

Comment:
2 things here:

1. Since the chevron is now a button, could you help check that the styling of the chevron (i.e. background color) remains the same as before? Do take note that the panel chevron is used in multiple places in TEAMMATES. One example where I found inconsistent styling is in "Instructor view session results (course-wide)" page:

Before:
<img width="1234" alt="Screen Shot 2023-09-14 at 12 14 14 AM" src="https://github.com/TEAMMATES/teammates/assets/25262042/3948e74a-bd9a-443a-bf1b-5bd677d5fd54">



Now:
<img width="1229" alt="Screen Shot 2023-09-14 at 12 08 52 AM" src="https://github.com/TEAMMATES/teammates/assets/25262042/47e67d79-8cd1-41b3-bb3e-14c87e522bce">

I think making the background transparent should fix this :)

2. For the `aria-label`, since this panel chevron is used in multiple pages of TEAMMATES, I think the current label is over specific? I think we can have a more general label like "Expand panel". What do you think?

Reply:
All of these make sense to me. I'll make the changes this evening :D